{
  try {
    testSuccessfulListUsersAction(subject,count);
    fail("Expected exception to be thrown");
  }
 catch (  QueryExecutionException e) {
    assertTrue("Exception should contain '" + AuthProcedures.PERMISSION_DENIED + "'",e.getMessage().contains(AuthProcedures.PERMISSION_DENIED));
  }
}
